src/theme/design-tokens/palette.ts

```diff
@@ -1,60 +1,122 @@
-// teal
-export const sb_green = '#00b3b0'
-export const sb_green_75 = '#40c6c4'
-export const sb_green_50 = '#7fd9d7'
-export const sb_green_25 = '#d9f4f3'
-
-// Ink
-export const sb_dark_blue = '#1b243f'
-export const sb_dark_blue_75 = '#545b6f'
-export const sb_dark_blue_50 = '#8d919f'
-export const sb_dark_blue_25 = '#c6c8cf'
-
-// green
-export const green = '#2db47d'
-export const green_75 = '#62c79e'
-export const green_50 = '#96d9be'
-export const green_25 = '#caecde'
-export const green_disabled = '#004e4c'
-// yellow
-export const yellow = '#fbce41'
-export const yellow_75 = '#fcdb71'
-export const yellow_50 = '#fde6a0'
-export const yellow_25 = '#fef3cf'
-
-// blue
-export const blue = '#395ece'
-export const blue_75 = '#6b87db'
-export const blue_50 = '#9caee6'
-export const blue_25 = '#cdd7f3'
-
-// orange
-export const orange = '#ffac00'
-export const orange_75 = '#ffc140'
-export const orange_50 = '#ffd57f'
-export const orange_25 = '#ffeabf'
-
-// red
-export const red = '#ff6159'
-export const red_75 = '#ff8983'
-export const red_50 = '#ffb0ac'
-export const red_25 = '#ffd7d5'
-
-// light
-export const light = '#dfe3e8'
-export const light_75 = '#e7eaee'
-export const light_50 = '#eff1f3'
-export const light_25 = '#f7f8f9'
-export const light_gray = '#b1b5be'
-
-export const black = '#101525'
-export const white = '#fff'
-export const black_trasparent = 'rgba($sb-dark-blue, 0.2)'
-
-export const color_primary = sb_green
-export const color_primary_dark = sb_dark_blue
-export const color_secondary = green
-export const color_positive = green
-export const color_negative = red
-export const color_warning = orange
-export const color_white = white
+// primary
+export const sb_primary_50 = '#d5fcfa'
+export const sb_primary_100 = '#b5ece9'
+export const sb_primary_200 = '#9ae6e2'
+export const sb_primary_300 = '#72dfda'
+export const sb_primary_400 = '#37dcd3'
+export const sb_primary_500 = '#04c8c0'
+export const sb_primary_600 = '#00b3b0'
+export const sb_primary_700 = '#05807f'
+export const sb_primary_800 = '#0a6465'
+export const sb_primary_900 = '#0a6465'
+export const sb_primary_950 = '#003033'
+export const sb_primary = sb_primary_700
+
+// secondary
+export const sb_secondary_50 = '#f7f9fc'
+export const sb_secondary_100 = '#dde3ee'
+export const sb_secondary_200 = '#b5c0e2'
+export const sb_secondary_300 = '#9eaacf'
+export const sb_secondary_400 = '#9eaacf'
+export const sb_secondary_500 = '#8794bc'
+export const sb_secondary_600 = '#5f6e9a'
+export const sb_secondary_700 = '#495781'
+export const sb_secondary_800 = '#424d6d'
+export const sb_secondary_900 = '#293455'
+export const sb_secondary_950 = '#1b243f'
+export const sb_secondary = sb_secondary_950
+
+// base
```

Comment:
What is meant by "base"

Reply:
kept the same pattern of names from design system https://storyfront-design-system-git-feat-shape-80-e56baa-storyblok-com.vercel.app/?path=/docs/foundations-colors--documentation

Reply:
I just map with the same name that is in design system and there this group of colors there's no description, at least until where I saw